src/US_Street/Lookup.php

```diff
@@ -51,12 +57,32 @@ public function __construct($street = null, $street2 = null, $secondary = null,
         $this->matchStrategy = $matchStrategy;
         $this->maxCandidates = $maxCandidates;
         $this->outputFormat = $outputFormat;
+        $this->countySource = $countySource;
+        $this->customParamArray = array();
+        $this->jsonArray = array(
```

Comment:
Why is the transitory serialization array stored on the Lookup object (`$this->jsonArray`)? This is particularly odd because it seems it would only be fully valid after `jsonSerialize` is called, and potentially invalid thereafter if any setters are called.

Reply:
I had other plans for jsonArray earlier in the development process to make it a more permanent object, but it ended up being replaced in that aspect by customParamArray. You are correct, that does not need to be a member of the Lookup object. That should be resolved in my latest commit. Thanks for pointing that out.